src/sentry/release_health/metrics.py

```diff
@@ -191,15 +192,19 @@ def _get_crash_free_rate_data(
 
         data: Dict[int, Dict[str, float]] = {}
 
-        session_status = resolve_tag_key(org_id, "session.status")
+        session_status = resolve_tag_key(UseCaseKey.RELEASE_HEALTH, org_id, "session.status")
 
         count_query = Query(
             match=Entity(EntityKey.MetricsCounters.value),
             select=[Function("sum", [Column("value")], "value")],
             where=[
                 Condition(Column("org_id"), Op.EQ, org_id),
                 Condition(Column("project_id"), Op.IN, project_ids),
-                Condition(Column("metric_id"), Op.EQ, resolve(org_id, SessionMRI.SESSION.value)),
+                Condition(
+                    Column("metric_id"),
+                    Op.EQ,
+                    resolve(UseCaseKey.RELEASE_HEALTH, org_id, SessionMRI.SESSION.value),
```

Comment:
Nit: Since this is all a release health module I would just instantiate it in the top of the module and just reuse it 
`USE_CASE = UseCaseKey.RELEASE_HEALTH`